packages/jitar-reflection/src/parser/Parser.ts

```diff
@@ -278,6 +278,16 @@ export default class Parser
         {
             return new ReflectionImport(members, token.value);
         }
+        else if (token.hasValue(Group.OPEN))
```

Comment:
why is there an else ?

Reply:
It's basically the same exception as the import above (without members), so it I found it logical to group them.